configs/providers.js

```diff
@@ -1,7 +1,7 @@
 module.exports = [
 	{
 		name: "RIBBON",
-		domain: "https://bins.ribbon.co",
+		domain: "http://bins.ribbon.co",
```

Comment:
Has ribbon really moved to http? If SSL is still an option we should favour https over http.